packages/smithy-aws-core/src/smithy_aws_core/__init__.py

```diff
@@ -1,12 +1,4 @@
-# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
-#
-# Licensed under the Apache License, Version 2.0 (the "License"). You
-# may not use this file except in compliance with the License. A copy of
-# the License is located at
-#
-#     http://aws.amazon.com/apache2.0/
-#
-# or in the "license" file accompanying this file. This file is
-# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
-# ANY KIND, either express or implied. See the License for the specific
-# language governing permissions and limitations under the License.
+#  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#  SPDX-License-Identifier: Apache-2.0
+
+__version__: str = "0.1.0"
```

Comment:
Is this 0.1.0 or 0.0.1? I think we've been using the latter everywhere else.

Reply:
I took this from the pyproject: https://github.com/smithy-lang/smithy-python/blob/develop/packages/smithy-aws-core/pyproject.toml#L3

Should those match?  Is there a convention in Python to keep these two values in sync?  (In Ruby for example the version is often written to a single file that both the code and library specification read from)

Reply:
Jonthan actually just brought this up, apparently we randomly bumped the version recently. I'm looking at getting it fixed, I don't think that was intentional. I'll follow up shortly.

Reply:
Can we use the [hatch version build hook](https://hatch.pypa.io/1.13/plugins/build-hook/version/) instead of maintaining version strings in init files?